cmd/provisioner-localpv/app/provisioner_hostpath.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright 2019 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+This file contains the volume creation and deletion handlers invoked by
+the github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/controller.
+
+The handler that are madatory to be implemented:
+
+- Provision - is called by controller to perform custom validation on the PVC
+  request and return a valid PV spec. The controller will create the PV object
+  using the spec passed to it and bind it to the PVC.
+
+- Delete - is called by controller to perform cleanup tasks on the PV before
+  deleting it.
+
+*/
+
+package app
+
+import (
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+
+	pvController "github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/controller"
+	mconfig "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	mPV "github.com/openebs/maya/pkg/kubernetes/persistentvolume/v1alpha1"
+	"k8s.io/api/core/v1"
+	//metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ProvisionHostPath is invoked by the Provisioner which expect HostPath PV
```

Comment:
Let us write code comments within 60 chars wherever possible.
It always nice/readable to write code that forms a column (remember newspapers columns)
This comment is not mine, but a much needed idiom.

Reply:
agreed. Will do.